VotingApplication/VotingApplication.Web/Api/Controllers/ManageController.cs

```diff
@@ -140,7 +155,30 @@ public void Put(Guid manageId, ManagePollUpdateRequest updateRequest)
                     newOptions.AddRange(updateRequest.Options);
                 }
 
+                List<Token> redundantTokens = existingPoll.Tokens.ToList<Token>();
```

Comment:
The whole put method is getting rather large and unwieldy. It'd help a lot if this was either broken down into a set of smaller, appropriately named methods, or extracted out into it's own class completely (which would make it a lot easier to test).